R/check_dhis2_attributes-helpers.R

```diff
@@ -0,0 +1,248 @@
+#' Get the dataset identifiers and names
+#'
+#' @param username the user name
+#' @param password the user's password
+#' @param base_url the base URL of the DHIS2 server
+#'
+#' @export
+#'
+get_data_sets <- function(base_url, username, password) {
+  checkmate::assertCharacter(base_url, len = 1L, null.ok = FALSE,
+                             any.missing = FALSE)
+  checkmate::assertCharacter(username, len = 1L, null.ok = FALSE,
+                             any.missing = FALSE)
+  checkmate::assertCharacter(password, len = 1L, null.ok = FALSE,
```

Comment:
Consider using `min.chars = 4L`. Assuming the lowest security that would be allowed in a real DHIS instance would not be less than a 4 digit PIN.

Reply:
will account for this